DokuwikiPage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This is the Dokuwiki export for FINDOLOGIC.
+ *
+ * If any bugs occur, please submit a new issue
+ * @see https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/issues/new
+ * @author Dominik Brader <support@findologic.com>
+ */
+
+class DokuwikiPage
+{
+    /**
+     * This is the time format as it gets outputted to the template.
+     * DAY.MONTH YEAR (HOUR:MINUTE)
+     */
+    const TIME_FORMAT = '%d.%B %Y (%H:%M)';
+
+    public $id;
+    public $url;
+    public $author;
+    public $lastEdit;
+    public $metadata;
+
+    /**
+     * Gets DokuWiki page data.
+     *
+     * * **id** - ID.
```

Comment:
Add field explanations to the fields as docblocks.

Reply:
:white_check_mark: done.